README.md

```diff
@@ -36,10 +36,39 @@ Interested in learning more about the basics of virtual environments in Python
 [article](http://www.cs.allegheny.edu/sites/gkapfham/programming/research/idea/2017/07/14/Virtual-Environments/)
 to further develop your understanding of this topic.
 
+GatorGrouper requires users to use Python 3. You can type `python --version` to check
+the current Python version on your device. If you do not have the right Python version,
```

Comment:
I'm not sure that "devices" is the best word. I wonder if "workstation" would be better? Or "development environment"? I think that "device" is too broad of a term.